ts/packages/cli/src/commands/export.ts

```diff
@@ -13,6 +24,21 @@ export const CONFIG_EXPORT_RUN_DEFAULT_ERROR_TEXT = 'could not export configurat
 
 type TemplateContext = { [key: string]: string } | { key: string; value: string }[];
 
+const CASE_FUNCTION: { [casing: string]: (str: string) => string } = {
+  CamelCase: camelCase,
+  CapitalCase: capitalCase,
+  ConstantCase: constantCase,
+  DotCase: dotCase,
+  KebabCase: kebabCase,
+  NoCase: noCase,
+  PascalCase: pascalCase,
+  PascalSnakeCase: (str: string) => capitalCase(str).split(' ').join('_'),
+  PathCase: pathCase,
+  SentenceCase: sentenceCase,
+  SnakeCase: snakeCase,
+  TrainCase: (str: string) => capitalCase(str).split(' ').join('-'),
+};
+
```

Comment:
NIT:
- VSCode doesn't like `str` as a parameter name due to the inbuilt spellchecker which causes unnecessary warnings, `string` would be preferable.
- I think this `Record<string, (string: string) => string>` is cleaner than `{ [casing: string]: (str: string) => string }`

Reply:
Sure